src/runtimes/docker/clients/DockerClientBase/DockerClientBase.ts

```diff
@@ -651,6 +654,42 @@ export abstract class DockerClientBase extends ConfigurableClient implements ICo
 
     //#endregion
 
+    //#region ImagesGenerateSbom Command
+
+    /**
+     * Generate the command line arguments for invoking a pull image command on
+     * a Docker-like client
+     * @param options Pull image command options
+     * @returns Command line arguments for pulling a container image
+     */
+    protected getGenerateSbomCommandArgs(options: ImageGenerateSbomCommandOptions): CommandLineArgs {
+        return composeArgs(
+            withArg('sbom'),
+            withNamedArg('--format', 'spdx-json', {shouldQuote: false}),

```

Comment:
We should probably let the user pick the appropriate format rather than hard coding spdx-json; at the very least we should add a format property to the command options and allow setting a specific output type as a vscode setting.

Reply:
I think of it more as informative than as part of a workflow--e.g., you do it to look at something, but not to copy and paste the output somewhere. So IMO it's fine to always use JSON. If a different format is desired to archive the file or similar, they should do so directly from the CLI.